cloudferrylib/os/storage/cinder_storage.py

```diff
@@ -533,7 +533,8 @@ def write_to_database(self, cmd, cursor, entries, primary_key=None):
             if 'snapshot_id' in entry:
                 entry.pop('snapshot_id')
             keys = entry.keys()
-            values = tuple([str(entry[k]) for k in keys])
+            values = tuple([str(entry[k]) if k is not None else "NULL"
```

Comment:
As results we "NULL" is a string in cinder DB, but it does not work. In case value is None need to pass None (smth like this: "if entry(k) is not None else None")